content/en/get-started/deploy-your-app.md

```diff
@@ -103,150 +114,71 @@ strategies:
 
-{{% alert title="Important" color="warning" %}}
-Armory CD-as-a-Service manages your Kubernetes deployments using ReplicaSet resources. During the initial deployment of your app, CD-as-a-Service deletes the underlying Kubernetes deployment object in a way that leaves behind the ReplicaSet and pods so that there is no actual downtime for your app. These are later deleted when the deployment succeeds.
-
-If your initial deployment fails, you should [manually delete]({{< ref "troubleshooting/tools#initial-deployment-failure-orphaned-replicaset" >}}) the orphaned ReplicaSet.
-{{% /alert %}}
-
-### Deploy first version
+### Deploy the first version
 
 1. Log in using the CLI.
 
    ```bash
    armory login
    ```
 
-   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process,.
+   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process.
 
-1. Start the deployment from the root of your directory.
+2. Start the deployment from the root of your directory.
 
    ```bash
    armory deploy start  -f deployment.yaml
    ```
 
-   The `deploy start` command starts your deployment. The command returns a **Deployment ID** and a link to your deployment details. 
+   This command starts your deployment, then returns a **Deployment ID** and a link to your deployment details. 
+
+3. Monitor your deployment execution.
 
-1. Monitor your deployment execution.
+   Use the link provided by the CLI to view your deployment in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments). 
 
-   Use the link provided by the CLI to navigate to your deployment details page in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments). 
+4. Issue manual approval.
 
-1. Issue manual approval.
+   Once CD-as-a-Service successfully deploys your resources to `staging`, it waits for your manual approval before deploying to `prod`. When the `staging` deployment has completed, click **Approve** to allow the `prod` deployment to begin. 
+   > You must issue manual approvals using the UI. You cannot issue manual approvals using the CLI.
 
-   Since you defined a manual approval to deploy to prod in your strategy, you need to issue your manual approval using the UI. Once the `staging` deployment has completed, click **Approve** to allow the `prod` deployment to begin. Because this is the first time deploying your app, CD-as-a-Service deploys 100% to your prod environment, skipping the defined `trafficSplit` strategy. CD-as-a-Service uses the `trafficSplit` when deploying subsequent versions of your app.
+  Because this is the first time deploying your app, CD-as-a-Service deploys 100% to your prod environment, skipping the defined `trafficSplit` strategy. CD-as-a-Service uses the `trafficSplit` when deploying subsequent versions of your app.
```

Comment:
Is this supposed to be part of #4? If yes, I think you need to add another space for the sentence to align.

![image](https://github.com/armory/docs-cdaas/assets/26799583/a75e89a3-455d-4516-9d86-b4c59b27f41f)